docs/state.md

```diff
@@ -179,6 +180,10 @@ gcsClient, err := storage.New(httpClient)
 
 ```
 
+## Scaleway (scw://)
+
+Scaleway storage is configured as a flavor of a S3 store.
```

Comment:
Add a documentation link to object storage: https://www.scaleway.com/en/docs/storage/object/quickstart/